### ISSUE TITLE
Training with OASIS

### ISSUE THREAD

**Issue by hpbzxxxx** [DiffuseMorph/DiffuseMorph#10]
I trainned the model with oasis dataset. The loss went wrong after several epochs of training. How did you configure the network and hyperparameters?
Thanks.
